mathics/core/convert/sympy.py

```diff
@@ -5,6 +5,7 @@
 Conversion to SymPy is handled directly in BaseElement descendants.
 """
 
+from collections.abc import Iterable
```

Comment:
The `Iterable` class that supports typing (`Iterable[sympy.Symbol]`) is in the `typing` module. You should also import `Dict`, `List` and `Set` for the following annotations.

Reply:
typing.Iterable is deprecated. https://docs.python.org/3/library/typing.html#typing.Iterable

Reply:
You are right. However, as we still support 3.8, we should do a conditional import.